docs/content/manual/_incoming/2642-remove-pod-scheduling.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Remove Pod Scheduling from harvester rke2 and rke1
+---
+
+* Related issues: [#2642](https://github.com/harvester/harvester/issues/2642) [BUG] Remove Pod Scheduling from harvester rke2 and rke1
+
+## Category: 
+* Rancher
+
+## Test Information
+Test Environment: 1 node harvester on local kvm machine
+Harvester version: v1.0-44fb5f1a-head (08/10)
```

Comment:
The version of Harvester and Rancher could be the current one and above, right?

Reply:
Yes. I was unsure on how best to show that in the test. I'll update it so that it's less ambiguous.